README.md

```diff
@@ -77,13 +80,50 @@ Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full l
    ```
    VITE_HYDRA_URL=ws://localhost:8929 npm start
    ```
-</details>
-
+   </details>
 
 ## CI/CD
 
 We practice continuous integration and deployment. Every branch and pull request triggers a build and deployment to an ephemeral environment. Merging to master causes a build and deployment to our [development environment](https://web.dev.reactivetrader.com).
 
+## Reactive Launcher
+
+<details open>
+<summary>Run Reactive Trader® Launcher locally</summary>
+
+Run:
+
+```
+npm run openfin:run:launcher
+```
+
+</details>
+
+Reactive Trader® Launcher interprets and executes tasks based on natural language commands expressed by the user.
+It serves as a convenient launchpad for a range of other Reactive applications, such as Reactive Trader FX (RTFX), Reactive Trader Credit (RTC), Reactive Analytics (RA), and Limit Checker (LC).
+
+The Reactive Trader® Launcher search engine supports a wide range of commands.
+Here are a few examples:
+
+```
+Buy USDJPY 10M
+How is Cable doing?
+How is the $currency market?
```

Comment:
`$currency` is not clear .. I'd just use a common example like GBPUSD .. there are 9 pairs in our standard set